test_dir/new_file_2

```diff
@@ -0,0 +1 @@
+well lets see what happens now
```

Comment:
@RiflerRick committed on a restricted repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) in the file test_dir/new_file_2